src/oadp_resources/oadp/datamover/volume_snapshot_backup.py

```diff
@@ -117,3 +119,7 @@ def wait_for_done(self, wait_timeout=240, sleep=5):
             sleep=sleep,
             wait_timeout=wait_timeout
         )
+
+    def log_status(self):
```

Comment:
Please move to utils,
it is relevant not only for this resource but all.

Reply:
True, thanks, I will patch it.

Reply:
Done.